phpBB/viewforum.php

```diff
@@ -47,27 +47,41 @@
 	trigger_error('NO_FORUM');
 }
 
+$sql_ary = array(
+	'SELECT'	=> "f.*",
+	'FROM'		=> array(
+		FORUMS_TABLE		=> 'f'
```

Comment:
You've specified the forums table here but further down on line 59 the forums table gets defined again in `$sql_form`. You need to clean up the code a bit more.